### DISCUSSION TITLE
Consider renaming `pole_density_function` to `pole_density_distribution`

### DISCUSSION THREAD

**Discussion by harripj** [pyxem/orix#447] [General]
I think perhaps it would be quite informative as to rename `pole_density_function` to `pole_density_distribution`, as in these functions we are computing the quantitative density distribution rather than internally creating a functional representation of the pole distribution. Perhaps the new function name would be more clear for the user and more representative for those inspecting the code.

**Comment by hakonanes** [comment]
Sounds good to me. We can deprecate the former in v0.12.0 and point to the latter, with removal in v0.13.0.
